updated definition of gist:Medium and added examples (#1293)
semanticarts/gist: docs/release_notes/issue1162-Medium-definition-updated.md

### Patch Updates

> - Updated the definition of `gist:Medium` to fix uncommon use of the word "physicality" and clarify overall meaning. Added `skos:example` with examples. Issue [#1162](https://github.com/semanticarts/gist/issues/1162).

Reviewer: ```suggestion
- Updated the definition of `gist:Medium` for clarity and added examples. Issue [#1162](https://github.com/semanticarts/gist/issues/1162).
```
